components/server/src/server.rs

```diff
@@ -1443,6 +1446,10 @@ where
             info!("disk space checker not enabled");
             return;
         }
+        let raft_path = engines.raft.get_engine_path().to_string();
+        let separate_raft_path = path_in_diff_mount_point(raft_path.as_str(), engines.kv.path());
```

Comment:
I think `separated_raft_disk_path` or `separated_raft_mount_path` is more proper.

Reply:
Fixed.